test/NuGet.Clients.Tests/NuGet.PackageManagement.VisualStudio.Test/ProjectSystems/LegacyPackageReferenceRestoreUtilityTests.cs

```diff
@@ -1958,6 +1958,161 @@ public async Task DependencyGraphRestoreUtility_LegacyPackageRef_Restore_BuildTr
+                    var testNuGetProjectContext = new TestNuGetProjectContext();
+                    var deleteOnRestartManager = new TestDeleteOnRestartManager();
+                    var nuGetPackageManager = new NuGetPackageManager(
+                        sourceRepositoryProvider,
+                        testSettings,
+                        testSolutionManager,
+                        deleteOnRestartManager);
+
+                    // set up projects
+                    var projectTargetFrameworkStr = "net45";
+                    var fullProjectPath = Path.Combine(testSolutionManager.TestDirectory, "Project2", "project2.csproj");
+                    var projectNames = new ProjectNames(
+                        fullName: fullProjectPath,
+                        uniqueName: Path.GetFileName(fullProjectPath),
+                        shortName: Path.GetFileNameWithoutExtension(fullProjectPath),
+                        customUniqueName: Path.GetFileName(fullProjectPath));
+                    var vsProjectAdapter = new TestVSProjectAdapter(
+                        fullProjectPath,
+                        projectNames,
+                        projectTargetFrameworkStr,
+                        restorePackagesWithLockFile: null,
+                        nuGetLockFilePath: null,
+                        restoreLockedMode:false,
+                        projectPackageVersions: new List<(string Id, string Version)>() {packageA, packageB });
+
+                    var projectServices = new TestProjectSystemServices();
+                    projectServices.SetupInstalledPackages(
+                        NuGetFramework.Parse(projectTargetFrameworkStr),
+                        new LibraryDependency
+                        {
+                            LibraryRange = new LibraryRange(
+                                packageA.PackageId,
+                                versionRange: null,
+                                LibraryDependencyTarget.Package),
+                        });
+
+                    var legacyPRProject = new LegacyPackageReferenceProject(
+                        vsProjectAdapter,
+                        Guid.NewGuid().ToString(),
+                        projectServices,
+                        _threadingService);
+                    testSolutionManager.NuGetProjects.Add(legacyPRProject);
+
+                    var packageContextB = new SimpleTestPackageContext(packageB.PackageId, packageB.Version);
+                    packageContextB.Files.Clear();
+                    packageContextB.AddFile("build/packageB.targets");
+                    packageContextB.AddFile("build/packageB.props");
+                    packageContextB.AddFile("buildCrossTargeting/packageB.targets");
+                    packageContextB.AddFile("buildCrossTargeting/packageB.props");
+                    packageContextB.AddFile("buildTransitive/packageB.targets");
```

Comment:
Are the targets and props files relevant to the test? I think it could be simplified a little if removed. same for the other package below.

Reply:
No - removed.